src/native/routes/index.js

```diff
@@ -27,8 +27,12 @@ import ProfileComponent from '../components/User/Profile';
 import AboutComponent from '../components/About';
 
 import ScannerComponent from '../components/Scanner/BarcodeScanner';
+import ButtonsComponent from '../components/Scanner/Buttons';
 
 
+/* TODO: Make Scanner Component open Buttons Component on successful
```

Comment:
You can use the router to switch between scenes by using Actions.name_of_key(component_params);

For example in Profile.js there's 
`<ListItem onPress={Actions.updateProfile} icon>
` which corresponds to  `        <Scene
          back
          key="updateProfile"
` in the router
So here you could make the ButtonsComponent take in the data that was just scanned, then call Actions.buttons(data) and it'll switch you to that scene. There's a good example in the README: https://github.com/aksonov/react-native-router-flux

Reply:
gotcha thanks